docs/03-pages/01-building-your-application/06-configuring/11-draft-mode.mdx

```diff
@@ -156,7 +156,9 @@ Then, send a request to `/api/disable-draft` to invoke the API Route. If calling
 
 ### Works with `getServerSideProps`
 
-Draft Mode works with `getServerSideProps` as well. It will also be available on the `context` object containing `draftMode`
+Draft Mode works with `getServerSideProps` as well. It will also be available on the `context` object containing `draftMode`.
+
+> **Good to know**: Avoid setting the `Cache-Control` header because it cannot be bypassed. Instead, use [ISR](/docs/pages/building-your-application/data-fetching/incremental-static-regeneration).
```

Comment:
I don't think we need to recommend avoiding setting it as it is valid to use still just not in combination with draft mode.

Reply:
Suggestion: 

```suggestion
> **Good to know**: You shouldn't set the `Cache-Control` header when using Draft Mode because it cannot be bypassed. Instead, we recommend using [ISR](/docs/pages/building-your-application/data-fetching/incremental-static-regeneration).
```